src/Sulu/Bundle/AdminBundle/Resources/js/containers/List/index.js

```diff
@@ -48,7 +48,7 @@ export {
     TableAdapter,
     FolderAdapter,
     FlatStructureStrategy,
-    PaginatedLoadingStrategy,
+    DefaultLoadingStrategy,
```

Comment:
```suggestion
    PaginatedLoadingStrategy,
    DefaultLoadingStrategy,
```

we still need to export the `PaginatedLoadingStrategy` for backwards compatibility 🙂